### ISSUE TITLE
Add article pagination to component content on docs.vf.io

### ISSUE THREAD

**Issue by kwm14** [canonical-web-and-design/vanilla-framework#2011]
**Describe the bug**
Add article pagination to components on docs.vf.io as once you've read about a particular component you may want to read the following one.

**To Reproduce**
1. Go to https://docs.vanillaframework.io
2. Click on Table side bar navigation
3. Scroll down to bottom of page
4. Next or previous patterns not available and you then have to scroll through the sidebar navigation to locate where you are

**Screenshots**
![screenshot 2018-10-02 at 16 52 58](https://user-images.githubusercontent.com/17748020/46360583-abbaad00-c663-11e8-8ecc-1c1d5b8ed049.png)

**Comment by bartaz** [comment]
This will involve finding solution for the docs to know what is next/prev documents. Possibly we should extract navigation structure (form hardcoded in template) to some data structure (json, yaml).

**Comment by matthewpaulthomas** [comment]
Named pagination links could have a drawback of boosting the search ranking of irrelevant search results. (Currently every pattern page is an irrelevant search result for the name of every other pattern, because of the always-visible table of contents on every page. But repeating the previous/next pattern names, nearer each page’s unique text, might make them rank higher.)

> once you've read about a particular component you may want to read the following one

We could use Google Analytics to determine whether people often do this currently. (I doubt it — except for “Heading icon”, “Icons”, “Images”, and “Inline images”, which probably should be a single page anyway.)

**Comment by lyubomir-popov** [comment]
@matthewpaulthomas those are interesting points, thanks Matthew. What is your gut feeling - is this a bad idea we should drop, or is it something we need to implement carefuly to avoid upsetting search ranking?

**Comment by matthewpaulthomas** [comment]
I suggested using Google Analytics as a way to avoid relying on gut feeling! But my guess is that pagination links would be seldom used here.

The original problem of “you then have to scroll through the sidebar navigation to locate where you are” has been fixed since this was reported. Visiting any component page still scrolls the sidebar needlessly, but at least it keeps the selected component visible.
